css/main.css

```diff
@@ -56,19 +61,177 @@ textarea {
    Author's custom styles
    ========================================================================== */
 
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+.black {
+  color: #000;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+}
+
+.main {
+  display: flex;
+  overflow: hidden;
+  font: normal 16px sans-serif;
+  width: 100%;
+  height: 100%;
+}
+
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  background: #0e171d;
+  color: #fff;
+  width: 300px;
+  height: 100%;
+  float: left;
```

Comment:
Minor note, but we're pretty set on ordering our CSS properties alphabetically. While it's silly, it helps on longer rule sets, and can prevent you from accidentally specifying the same property twice.